src/halo-hooks/useRewards.ts

```diff
@@ -9,8 +9,8 @@ import { ZERO_ADDRESS } from '../constants'
 import { BigNumber } from 'ethers'
 import { PENDING_REWARD_FAILED } from 'constants/pools'
 
-export const useLPTokenAddresses = () => {
-  const rewardsContract = useHALORewardsContract()
+export const useLPTokenAddresses = (rewardsVersion = 1) => {
```

Comment:
Should we do something like this?
```suggestion
export const useLPTokenAddresses = (rewardsVersion : number = 1) => {
```

Reply:
Pls see https://github.com/HaloDAO/halodao-interface/pull/61#discussion_r704894614